src/appmixer/hubspot/crm/CreateContact/CreateContact.js

```
@@ -48,7 +55,11 @@ module.exports = {
             phone: properties.phone ? properties.phone : '',
             state: properties.state ? properties.state : '',
             address: properties.address ? properties.address : '',
-            email: properties.email ? properties.email : ''
+            email: properties.email ? properties.email : '',
+            company: properties.company ? properties.company : '',
+            ...Object.entries(properties)
+                .filter(([key]) => Object.keys(customProperties).includes(key))
+                .reduce((acc, [key, value]) => ({ ...acc, [key]: value || '' }), {})
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)